src/Symfony/Component/Routing/Loader/AttributeClassLoader.php

```diff
@@ -107,6 +107,15 @@ public function load(mixed $class, ?string $type = null): RouteCollection
             return $collection;
         }
         $fqcnAlias = false;
+
+        if (!$class->hasMethod('__invoke')) {
+            foreach ($this->getAttributes($class) as $attr) {
+                if ($attr->getAliases()) {
+                    throw new \InvalidArgumentException(\sprintf('Route aliases cannot be used on non-invokable class "%s".', $class->getName()));
```

Comment:
```suggestion
                    throw new InvalidArgumentException(\sprintf('Route aliases cannot be used on non-invokable class "%s".', $class->getName()));
```

And do the proper import from Routing exception namespace.